src/containers/flows/MintDropNFT/labels.tsx

```diff
@@ -37,12 +37,33 @@ export const getBannerLabel = (
 	}
 	if (dropStage === Stage.Whitelist) {
 		if (isFinished) {
-			<>Wilder Pets public release starting - you may need to refresh</>;
+			<>Wilder Beasts public release starting - you may need to refresh</>;
 		} else {
+			if (countdownDate && !isFinished) {
+				return (
+					<div style={{ display: 'flex', flexDirection: 'column' }}>
+						<span>
+							Available to everyone in{' '}
+							{countdownDate && (
+								<Countdown to={countdownDate} onFinish={onFinish} />
+							)}
+						</span>
+					</div>
+				);
+			}
+			if (countdownDate && isFinished) {
+				return (
+					<div style={{ display: 'flex', flexDirection: 'column' }}>
+						<span>
+							Public release starting now - waiting for contract to begin
+						</span>
+					</div>
+				);
+			}
 			return (
 				<div style={{ display: 'flex', flexDirection: 'column' }}>
 					<span>
-						Wilder Pets now available for whitelisted supporters{' '}
+						Presale Minting is now Open!{' '}
```

Comment:
If someone has specifically said to use `Open` with a capital `o` then go for it, same with the `m` in `Minting`. If not, change to `Presale minting is now open!`